mapbox/libjava-services/src/main/java/com/mapbox/services/api/directions/v5/MapboxDirections.java

```diff
@@ -578,5 +575,25 @@ public MapboxDirections build() throws ServicesException {
       }
       return new MapboxDirections(this);
     }
+
```

Comment:
@zugaldia Since I don't know in depth the SDK, does it mean that `coordinates` would be a list of coordinates in between the `origin` and the `destination`? At the moment, `origin` is the first element of the list and `destination` is the last one. When you try to add a new `origin`, if you already had more than two `coordinates`, the new `origin` is prepended to the `coordinates` list. Similarly, when you try to add a new `destination`, if you already had more than two `coordinates`, the new `destination` is appended. But, if you called `setCoordinates` the whole `coordinates` is reset. Is that the behavior that we are looking for?
cc/ @cammace